Hyperlinks for RI highlights (#616)
zowe/cics-for-zowe-client: packages/vsce/CHANGELOG.md

All notable changes to the "cics-extension-for-zowe" extension will be documented in this file.

> ## Recent Changes

Reviewer: Yeah changelog here would be great - I think there was one previously, I commented on it!

## `3.21.0`

- Enhancement: Added "Show in Data sets View" command for CICS Library Datasets to view them in Zowe Explorer. [#633](https://github.com/zowe/cics-for-zowe-client/issues/633)
- Enhancement: Inquire program, Show Library and Inquire Transaction can open respective resources from Resource Inspector. [#449](https://github.com/zowe/cics-for-zowe-client/issues/449)
- BugFix: Region filtering selects only regions. [#575](https://github.com/zowe/cics-for-zowe-client/issues/575)
- BugFix: Enable previously used resource filters to be edited before reuse. [#618](https://github.com/zowe/cics-for-zowe-client/issues/618)

> 

Reviewer: ```suggestion
```

Author: fixed as suggested

## Recent Changes

- Enhancement: Inquire program, Show Library and Inquire Transaction can open respective resources from Resource Inspector. [#449](https://github.com/zowe/cics-for-zowe-client/issues/449)

> - Enhancement: Datasets,USS and logs will have hyperlink in RI highlights. [#621](https://github.com/zowe/cics-for-zowe-client/issues/621)

Reviewer: ```suggestion
- Enhancement: Adds hyperlinks to appropriate attributes in RI highlights. [#621](https://github.com/zowe/cics-for-zowe-client/issues/621)
```

Author: fixed as suggested
